R/wwinference.R

```diff
@@ -31,13 +31,10 @@
 #' `get_model_spec()`. The default here pertains to the `forecast_date` in the
 #' example data provided by the package, but this should be specified by the
 #' user based on the date they are producing a forecast
-#' @param fit_opts The fit options, which in this case default to the
-#' MCMC parameters as defined using `get_mcmc_options()`. This includes
-#' the following arguments, which are passed to
-#' [`$sample()`][cmdstanr::model-method-sample]:
-#' the number of chains, the number of warmup
-#' and sampling iterations, the maximum tree depth, the average acceptance
-#' probability, and the stan PRNG seed
+#' @param fit_opts The fit options to pass to
+#' [`$sample()`][cmdstanr::model-method-sample], must be a valid cmdstanr
+#' sample arg. Default is an empty list, which uses the
+#' default parameters in `get_mcmc_options()`
```

Comment:
```suggestion
#' @param fit_opts MCMC fitting options, as a list of keys and values. 
#' These are passed as keyword arguments to [`$sample()`][cmdstanr::model-method-sample].
#' Where no option is specified, [wwinference()] will fall back first on a package-specific default value 
#' defined by [get_mcmc_options()]. If no package-specific default exists, it will fall back on
#' the default value defined in [`$sample()`][cmdstanr::model-method-sample].
```